### ISSUE TITLE
Minor issue, no viable alternative at input when there is a define at end of include file

### ISSUE THREAD

**Issue by cverbiest** [Riverside-Software/sonar-openedge#811]
The issue is minor but it took me some time to find the cause. 
Sample code : [src.zip](https://github.com/Riverside-Software/sonar-openedge/files/4919200/src.zip)

```
Syntax error -- <unknown> -- src\ifoption.i:1:5 -- no viable alternative at input 'if (yesﾙ'
Error during code parsing for [uri=file:src/custom.p] at position src\ifoption.i:1:5
```

Issue can be worked-around by adding a newline to the end of myoptions.i

**Comment by gquerret** [comment]
I'll have a look at the code, thanks !

**Comment by gquerret** [comment]
Regression introduced in 2.8 (commit af0bd22). Fixed in develop, will be included in 2.9.
